### ISSUE TITLE
South Korea Supersonic Anti ship Missile

### ISSUE THREAD

**Issue by clerara10** [PygmalionOfCyprus/cmo-db-requests#4322]
### DB Selector

DB3K

### Baseline

SSC-5 Stooge [P-800 Onyx] (2012)

### Hypothetical

No

### Name

Korea Supersonic Anti ship Missile

### Length (m)

6.6

### Width/Span (m)

1.4

### Diameter (m)

0.53

### Weight (kg)

1500

### Range (nm)

323

### Target Altitude / Depth (ft)

Same as P-800 Onyx

### Target Speed (kts)

Same as P-800 Onyx

### Launch Altitude / Depth (ft)

Same as P-800 Onyx

### LaunchSpeed (kts)

Same as P-800 Onyx

### Warhead

300kg HE

### Targets

Surface Vessel, Land Structure - Soft, Land Structure - Hardened, Runway

### CEP (m)

Same as P-800 Onyx

### Sensors

1x Ku band Active Radar
1x IR seeker
1x Passive Radar Seeker

### Weapon Codes

Home On Jam (HOJ), Capable vs Seaskimmer, Terrain Following, Search Pattern, Bearing-Only Launch (BOL), Re-Attack Capability, Uses GPS, Level Cruise Flight

### Comments

The Korea Supersonic Anti-ship Missile is a supersonic anti-ship missile developed by South Korea based on the technology of the P-800 Onyx. Released in 2021, its official name has yet to be made public and is being called the Korean version of the supersonic anti-ship missile. By applying the stealth design, it achieved a lower RCS level than the P-800 and It also reduced the probability of being detected by the enemy through sea skimming flights

It can be launched from missile sites on the ground and KVLS and has been deployed for combat in 2022

### Sources

https://www.navalnews.com/naval-news/2021/09/south-korea-unveils-new-supersonic-anti-ship-missile/

https://www.youtube.com/watch?v=6G22cyimEs4

https://en.yna.co.kr/view/AEN20210915010951325
![South-Korea-Unveils-New-Supersonic-Anti-Ship-Missile-1024x547](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/66541619/a0fcb0f9-be76-4f0f-a401-5e1679e2cba1)
![South_Korea_has_developed_indigenous_SLBM_Submarine-Launched_Ballistic_Missile_925_001](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/66541619/5c8df9a1-1d49-4788-b350-1a15898428c1)
![novaya-yuzhnokoreiskaya-sverhzvukovaya-protivokorabelnaya-raketa-uin9gabh-1631956862](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/66541619/cdc9e8bb-bd1b-41bf-91ad-6aa43ca7d58e)

**Comment by LuigiP57** [comment]
Should we give it a/some platform/s? You said it could be mounted in KVLS.

**Comment by LuigiP57** [comment]
And what do you think of HyCore?

**Comment by clerara10** [comment]
> And what do you think of HyCore?

The Supersonic Anti-ship Missile can be mounted on KVLS and can be operated on ships equipped with KVLS. Hycore is also a separate weapon as a Korean Hypersonic weapon prototype

**Comment by LuigiP57** [comment]
> The Supersonic Anti-ship Missile can be mounted on KVLS and can be operated on ships equipped with KVLS. Hycore is also a separate weapon as a Korean Hypersonic weapon prototype

Maybe you misunderstood me. As for "platform", I thought maybe you could add some specific platforms for DB manager to refer to. As for "HyCore", I once wanted to propose a Hypo classification ticket, but the lack of information made me give up the idea. I wonder if you have more information about HyCore?

**Comment by clerara10** [comment]
> > The Supersonic Anti-ship Missile can be mounted on KVLS and can be operated on ships equipped with KVLS. Hycore is also a separate weapon as a Korean Hypersonic weapon prototype
> 
> Maybe you misunderstood me. As for "platform", I thought maybe you could add some specific platforms for DB manager to refer to. As for "HyCore", I once wanted to propose a Hypo classification ticket, but the lack of information made me give up the idea. I wonder if you have more information about HyCore?

It literally operates on KVLS-equipped ships and ground-based TELs. For example, it can be operated on Jeongjo the Great class destroyer, Sejong the Great class, and Yi Sun-shin class destroyer. Hycore is the name of the Korea Hypersonic Missile test projectile. For example, with the JSF program, it's like the X-35. I don't think it's appropriate to put it as a Hypo ticket

**Comment by LuigiP57** [comment]
> For example, it can be operated on Jeongjo the Great class destroyer, Sejong the Great class, and Yi Sun-shin class destroyer.

Yes, I mean giving DBIDs of the potential platforms directly makes it easier for the DB manager to add weapons to them. Instead of having no platform like the previous AIM-120D-3 and PL-21.

> Hycore is the name of the Korea Hypersonic Missile test projectile. For example, with the JSF program, it's like the X-35. I don't think it's appropriate to put it as a Hypo ticket.

What I want to know is, do we have more technical details on HyCore? If not, then we can only give a speculative range, warhead etc. Therefore I would say that it is better to classify it as Hypo.

**Comment by LuigiP57** [comment]
> and has been deployed for combat in 2022

Forgive me, I can't seem to find any information to support this, or see it in the articles you quoted. May I ask how this conclusion was reached?

**Comment by clerara10** [comment]
> > and has been deployed for combat in 2022
> 
> Forgive me, I can't seem to find any information to support this, or see it in the articles you quoted. May I ask how this conclusion was reached?

ccording to the Korean news, the Minister of Defense confirmed that the actual deployment has been made.
http://news.heraldcorp.com/view.php?ud=20220228000565

The Supersonic Cruise Missile in the article is Supersonic Antiship Missile. It's not officially called Antiship Missile, but it's actually Antiship Missile.

**Comment by clerara10** [comment]
> > For example, it can be operated on Jeongjo the Great class destroyer, Sejong the Great class, and Yi Sun-shin class destroyer.
> 
> Yes, I mean giving DBIDs of the potential platforms directly makes it easier for the DB manager to add weapons to them. Instead of having no platform like the previous AIM-120D-3 and PL-21.
> 
> > Hycore is the name of the Korea Hypersonic Missile test projectile. For example, with the JSF program, it's like the X-35. I don't think it's appropriate to put it as a Hypo ticket.
> 
> What I want to know is, do we have more technical details on HyCore? If not, then we can only give a speculative range, warhead etc. Therefore I would say that it is better to classify it as Hypo.

I can't access the game for the time being, so I can't check my DBID.
Also, we don't know the technical details of Hycore

**Comment by claudejdev** [comment]
> I can't access the game for the time being, so I can't check my DBID. Also, we don't know the technical details of Hycore

In such a situation, this website can be useful @clerara10 :
https://cmano-db.com/
